### PR TITLE
model/add_unit_test_search_class

### PR DESCRIPTION
**Pull request by sofiacespedes**
Added Search class

### CODE REVIEW

File: src/java/com/jalasoft/search/model/Search.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
+/*
```
Comment (RafaelAlfaro) [inline src/java/com/jalasoft/search/model/Search.java]: Please update the Copyright with this information:
/*
 * Search .java
 *
 * Copyright (c) 2018 Jalasoft.
 *
 * This software is the confidential and proprietary information of Jalasoft.
 * ("Confidential Information").  You shall not
 * disclose such Confidential Information and shall use it only in
 * accordance with the terms of the license agreement you entered into
 * with Jalasoft.
 */

File: src/java/com/jalasoft/search/model/Search.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+ * Copyright (c) 2018 Jalasoft.
+ *
+ * This software is the confidential and propietary information of Jalasoft.
+ */
+package java.com.jalasoft.search.model;
+/**
+ * <Class description goes here>
```
Comment (RafaelAlfaro) [inline src/java/com/jalasoft/search/model/Search.java]: Please add a description related to the class.

File: src/java/com/jalasoft/search/model/Search.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+ */
+import java.io.File;
+
+public class Search {
+    public void Search() {
+
+    }
+
+    public void ListFileByPath(String name, File file) {
+        File[] listOfFiles = file.listFiles();
```
Comment (RafaelAlfaro) [inline src/java/com/jalasoft/search/model/Search.java]: Please add Documentation related to the method. 
e.g:
    /**
     * Verifies if the path is valid
     *
     * @return boolean
     */

File: src/java/com/jalasoft/search/model/Search.java [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+        if (listOfFiles != null)
+            for (File fil : listOfFiles) {
+                if (fil.isDirectory()) {
+                    ListFileByPath(name, fil);
+                } else if (name.equalsIgnoreCase(fil.getName())) {
+                    System.out.println(fil.getAbsolutePath());
+                }
+            }
+    }
+}
```
Comment (RafaelAlfaro) [inline src/java/com/jalasoft/search/model/Search.java]: Please add an "Enter" key. The file should finish with a new line.

**Comment by Ronalinc** [comment]
Please remove all comments inserted

**Comment by sofiacespedes** [comment]
closing because there were multiple conflicts

**Comment by sofiacespedes** [comment]
reopen it

**Review by RafaelAlfaro — APPROVED** [review]
Works for me. 👍

**Review by RafaelAlfaro — APPROVED** [review]
it is OK for me

**Pull request merged**